skopt/optimizer/optimizer.py

```diff
@@ -370,24 +388,40 @@ def tell(self, x, y, fit=True):
             only be fitted after `n_initial_points` points have been told to the
             optimizer irrespective of the value of `fit`.
         """
+        check_x_in_space(x, self.space)
+
+        if "ps" in self.acq_func:
+            if is_2Dlistlike(x):
+                if np.ndim(y) == 2 and np.shape(y)[1] == 2:
+                    y = [[val, log(t)] for (val, t) in y]
```

Comment:
A bit of a nitpick: maybe use log(t + 1e-3), for example in case user measures time in minutes, and then it might happen that for some task the time is zero minutes (< 60 seconds).